Resources/Prototypes/GameRules/events.yml

```diff
@@ -158,7 +158,7 @@
   id: ZombieOutbreak
   config:
     !type:StationEventRuleConfiguration
-    id: ZombieOutbreak
+    id: Zombie
```

Comment:
this is supposed to be the same as the gamerule ID

Reply:
..or, okay, this is weirder, i thought this handled both. this isnt really how its intended for this to work, but I guess it works

Reply:
> ..or, okay, this is weirder, i thought this handled both. this isnt really how its intended for this to work, but I guess it works

This was something i was aware of when adding event rule configs and it's intended
and no, it's not *great*, but it beats refactoring every rule to support  running multiple times at once

Reply:
Should i just leave it for now?